src/components/skeleton/style.css

```diff
@@ -8,7 +8,7 @@
 	width: var(--width);
 	height: 100%;
 	border-radius: 0.4rem;
-	background-image: linear-gradient(90deg, rgb(var(--color, var(--smoothly-dark-color))) 0, transparent 2.5rem, rgb(var(--color, var(--smoothly-dark-color))) 5rem);
+	background-image: linear-gradient(90deg, rgb(var(--text-color, var(--smoothly-dark-color))) 0, transparent 2.5rem, rgb(var(--text-color, var(--smoothly-dark-color))) 5rem);
```

Comment:
is this intended?

Reply:
No.

Reverting the file to its original state.